apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -68,6 +110,23 @@ router.get(
 router.post(
   '/:unsafe_lti13_course_instance_id',
   asyncHandler(async (req, res) => {
+    const instance = await queryRow(
+      sql.select_lti13_instance,
+      {
+        course_instance_id: res.locals.course_instance.id,
+        lti13_course_instance_id: req.params.unsafe_lti13_course_instance_id,
+      },
+      Lti13CombinedInstanceSchema,
+    );
+
+    const serverJobOptions = {
+      courseInstanceId: res.locals.course_instance.id,
+      userId: res.locals.user.user_id,
+      authnUserId: res.locals.authn_user.user_id,
+      type: 'lti13',
```

Comment:
Nit: consider using more precise job names so that we can e.g. show a history of all syncs, or something like that. That is, differentiate from syncs, create-and-link, etc.